lms/services/user.py

```diff
@@ -70,13 +73,13 @@ def upsert_user(self, lti_user: LTIUser) -> User:
         user.roles = lti_user.roles
         user.display_name = lti_user.display_name
         if lti_user.is_instructor:
-            # We are only storing emails for teachers now.
+            # Always store instructor emails
+            user.email = lti_user.email
+        elif lti_user.is_learner and ai_settings.get_setting(
+            ai_settings.fields[ai_settings.Settings.HYPOTHESIS_COLLECT_STUDENT_EMAILS]
+        ):
+            # Only store student emails if the feature flag is enabled
             user.email = lti_user.email
```

Comment:
Since both conditions have the same body (`user.email = lti_user.email`), should we combine with conditions with `or`?

Reply:
Yes! I did that on the roster side but here I was distracted by the existing condition